src/multivariate/powerspherical.jl

```diff
@@ -0,0 +1,77 @@
+"""
+The Power Spherical distribution is useful as a replacement for the von Mises-Fisher distribution.
+
+The probability density function of the Power Spherical distribution parameterized by the mean direction ``\\mu`` and the concentration parameter ``\\kappa`` is given by
+    
+```math
+p_{X}(x ; \\mu, \\kappa)= \\left\\{2^{\\alpha+\\beta} \\pi^{\\beta} \\frac{\\Gamma(\\alpha)}{\\Gamma(\\alpha+\\beta)}\\right\\}^{-1}\\left(1+\\mu^{\\top} x\\right)^{\\kappa}
+```
+"""
+struct PowerSpherical{T <: Real} <: ContinuousMultivariateDistribution
+    μ::Vector{T}
+    κ::T
+
+    function PowerSpherical(μ::Vector{T}, κ::T = one(T); checknorm = true, normalize_μ = true) where {T <: Real}
+        # check arguments
+        κ > 0 || error("κ must be positive.")
+        normalize_μ || !checknorm || isunitvec(μ) || error("μ must be a unit vector")
+             
+        new{eltype(μ)}(normalize_μ ? normalize(μ) : μ, κ)
+    end
+end
+
+function PowerSpherical(μ::Vector{T}, κ::Real; kwargs...) where {T<:Real}
+    R = promote_type(T, eltype(κ))
+    return PowerSpherical(convert(AbstractArray{R}, μ), convert(R, κ), kwargs...)
+end
+
+### Basic properties
+Base.length(d::PowerSpherical) = length(d.μ)
+Base.eltype(d::PowerSpherical) = eltype(d.μ)
+meandir(d::PowerSpherical) = d.μ
+concentration(d::PowerSpherical) = d.κ
+insupport(::PowerSpherical, x::AbstractVector{T}) where {T<:Real} = isunitvec(x)
+
+function sampler(d::PowerSpherical)
```

Comment:
Nope. Tbh, I don't think it makes much of a difference in terms of performance but it keeps the distribution itself lean and provides a nice separation between the parameters and the sampling. Otherwise we would need to store both a `Beta` and a `HyperSphericalUniform` int the `PowerSpherical` type.

Reply:
> Otherwise we would need to store both a Beta and a HyperSphericalUniform in the PowerSpherical type.

The usual alternative would rather be to construct these distributions in `_rand!`.